### ISSUE TITLE
fails to display on my ebook reader (Sony PRS-T2)

### ISSUE THREAD

**Issue by anarcat** [bookieio/Bookie#459]
any bookie website page totally fails to load on my e-book reader. it seems like the javascript never kicks in, even though it is enabled in the browser.

here's a picture of what it looks like:

![img_4659](https://cloud.githubusercontent.com/assets/796623/3048599/3e8ccea4-e14b-11e3-9f05-927bb98bbe28.JPG)

this is a pretty dumb device, so I am not sure how much this can be diagnosed... this is the device:

https://en.wikipedia.org/wiki/Sony_PRS-T2#2012_Model_.28Discontinued_late_2013.29

it says it runs a "heavily modified version of android"...

**Comment by mitechie** [comment]
Wow, that's interesting. I'll see if I can find any details on the browser. I'm not sure how far I can get with something like this, but I think it's really interesting.

**Comment by anarcat** [comment]
yeah, i was a little disappointed, but then again i am being more and more disapppointed with this locked-down device...
